tools/wheels/cibw_test_command.sh

```diff
@@ -26,5 +26,7 @@ fi
 # Set available memory value to avoid OOM problems on aarch64.
 # See gh-22418.
 export NPY_AVAILABLE_MEM="4 GB"
-python -c "import sys; import numpy; sys.exit(not numpy.test(label='full'))"
+# Run full tests with -n=auto. This makes pytest-xdist distribute tests across
+# the available N CPU cores: 2 by default for Linux instances and 4 for macOS arm64
+python -c "import sys; import numpy; sys.exit(not numpy.test(label='full', extra_argv=['-n=auto']))"
```

Comment:
Are you sure it is worthwhile distributing tests when there are only 2 cores? Does the total time go down?

Reply:
yes, it's faster. typically ~50% or so